fpdf/fpdf.py

```diff
@@ -3408,7 +3400,16 @@ def normalize_text(self, txt):
         # - for TTF unicode fonts: unicode object (utf8 encoding)
         # - for built-in fonts: string instances (encoding: latin-1, cp1252)
         if not self.unifontsubset and self.core_fonts_encoding:
-            return txt.encode(self.core_fonts_encoding).decode("latin-1")
+            try:
+                return txt.encode(self.core_fonts_encoding).decode("latin-1")
+            except UnicodeEncodeError as e:
```

Comment:
We try to avoid one-letter variable names: could you please rename this as `error`?